interface/themes/style_manila.css

```diff
@@ -0,0 +1,1897 @@
+/*! This style sheet was autogenerated using gulp + scss
```

Comment:
Note this style_manila.css file no longer exists. Are you needing to modify the manila theme for your work?

Reply:
But I thought the sass file was built off of this then run through gulp.
When in a docker I change this style_manilla.css file and reload it with npn run dev.
So is it really gone?  It was the only style sheet with "non-flat" styling IMHO.

Reply:
manila style is still there, but in sass form. the .css file gets built in the build process.

Reply:
hi @zbig01 ,
Was hoping you could help provide guidance for @ophthal on best way to get his styling changes into the scss form. -brady